test/salt/pillar/modules.sls

```diff
@@ -427,6 +427,9 @@ apache:
     RemoteIPTrustedProxy:
       - 10.0.8.0/24
       - 127.0.0.1
+    RemoteIPInternalProxy
```

Comment:
This too.

Reply:
Updated

Reply:
Thanks. Please update commit messages so https://github.com/semantic-release/semantic-release works.  I suggested "**feat(trusted_proxy):** Add support for RemoteIPInternalProxy" and 2nd could be "**fix(syntax):** add semicolon".